### PR TITLE
Change routes

### PR DESCRIPTION
**Pull request by KHafner**
# Pull Request Template

## Checklist:

- [x] I have written test cases for this change, and my code passed the test
- [x]  I have updated documentation to reflect this change
- [x] I have attached this PR to an issue/user story

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -206,44 +206,50 @@ def get_shopcart_items(shopcart_id):
...
     This endpoint will create a ShopcartItem based the data in the body that is posted
     """
     logger.info("Request to create a shopcart item")
     check_content_type("application/json")
     shopcart_item = ShopcartItem()
-    print(request.get_json)
     shopcart_item.deserialize(request.get_json())
+    if shopcart_item.sid != shopcart_id:
+        raise BadRequest("Shopcart id '{}' does not match sid in request '{}'.".format(
+                          shopcart_id, shopcart_item.sid))
```
Comment (iLtc) [inline service/routes.py:248]: If `shopcart_id` is in the URL, do we really want `sid` in the request body?

**Review by iLtc — APPROVED** [review]
LGTM, thanks!

File: service/routes.py (hunk 1 above)
Comment (KHafner) [reply service/routes.py]: Good point, I changed it to set it to the id contained in the URL

**Review by nonococoleo — APPROVED** [review]
great

**Pull request merged**